gsa/src/web/pages/extras/__tests__/feedstatuspage.js

```diff
@@ -17,17 +17,22 @@
  */
 import React from 'react';
 
-import {rendererWith, wait} from 'web/utils/testing';
+import {rendererWith, wait, waitForElement} from 'web/utils/testing';
 
 import FeedStatus from '../feedstatuspage';
 import {Feed} from 'gmp/commands/feedstatus';
 
 import Response from 'gmp/http/response';
 
+const mockDate = new Date(1595833720475);
+
+// set mockDate so the feed ages don't keep changing
+global.Date.now = jest.fn(() => mockDate);
```

Comment:
This needs to be reset after the test, or the .now method will be "permanently" overwritten.